jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/ResponseHttpFields.java

```diff
@@ -150,7 +150,7 @@ public HttpField next()
             public void remove()
             {
                 if (_committed.get())
-                    throw new UnsupportedOperationException("Read Only");
+                    return;
```

Comment:
For the jetty-core version of the response handling, I like the UnsupportedOperationException.
The choice to ignore the behavior is really a servlet specific behavior, not jetty-core

Reply:
I don't disagree, but when I tried that change it broke a few more tests and would need another wrapper abstraction to make EE9 work.   
Perhaps that is a 12.1 change?